src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements AreaCalculator {
```

Comment:
```suggestion
public class IsoscelesTrapezoid extends Figure {
```
same for other classes